Src/BootCamp.Chapter/Item.cs

```diff
@@ -3,15 +3,19 @@
     public class Item
     {
         private string _name;
```

Comment:
you can remove the private backing field and just use auto-implemented properties

Reply:
As recommended, I have removed the private backing field. and am using the auto-implemented properities.